scripts/scr_ui_advisors/scr_ui_advisors.gml

```diff
@@ -513,110 +605,218 @@ function scr_ui_advisors() {
 	        if (fest_type="Chapter Relic") then fet_text="Upgrade Existing";
-	        if (fest_type="Imperial Mass"){fet_text="Request Sororitas";fet_scale=0.75;}
 	        if (fest_type="Chapter Sermon") then fet_text="Drugs";
-	        if (fest_type="Triumphal March"){ fet_text="Brandish Bloody Trophies";fet_scale=0.6;}
+	        if (fest_type="Tournament"){
+				fet_text="Invite Other Chapters";
+				fet_scale=0.75;
+			}
+	        if (fest_type="Deathmatch"){
+				fet_text="Allow Other Competitors";
+				fet_scale=0.7;
+			}
+	        if (fest_type="Imperial Mass"){
+				fet_text="Request Sororitas";
+				fet_scale=0.75;
+			}
+	        if (fest_type="Triumphal March"){ 
+				fet_text="Brandish Bloody Trophies";
+				fet_scale=0.6;
+			}
         
-	        if (fest_feature3=0) then che=1;if (fest_feature3=1) then che=2;
+	        if (fest_feature3=0) then che=1;
+			if (fest_feature3=1) then che=2;
 	        if (fest_type="Imperial Mass") and (known[5]=0) then draw_set_alpha(0.5);
-	        cx=xx+1388+22;cy=yy+326;draw_text_transformed(cx,cy,string_hash_to_newline(string(fet_text)),fet_scale,1,0);cx-=35;cy-=4;
-	        draw_sprite(spr_creation_check,che+1,cx,cy);draw_set_alpha(1);
+
+	        cx=xx+1388+22;
+			cy=yy+326;
+			
+			draw_text_transformed(cx,cy,string_hash_to_newline(string(fet_text)),fet_scale,1,0);cx-=35;cy-=4;
+	        draw_sprite(spr_creation_check,che+1,cx,cy);
+			draw_set_alpha(1);
 	        // if (scr_hit(cx+31,cy,cx+260,cy+20)=true){tool1="Planet";tool2="Allows the use of vehicles, and bikes, but prevents this formation from being used during Raids.";}
-	        if (scr_hit(cx,cy,cx+32,cy+32)=true) and (mouse_left=1) and (cooldown<=0) and (dropdown_open=0){var onceh;onceh=0;cooldown=8000;
+	        if (scr_hit(cx,cy,cx+32,cy+32)=true) and (mouse_left=1) and (cooldown<=0) and (dropdown_open=0){
+				var onceh=0;
+				cooldown=8000;
 	            if (fest_type="Imperial Mass") and (known[5]=0) then onceh=1;
-	            if (onceh=0) and (fest_feature3=0){onceh=1;fest_feature3=1;}
-	            if (onceh=0) and (fest_feature3=1) and (fest_type!="Chapter Relic"){onceh=1;fest_feature3=0;}
-	            if (fest_type="Chapter Relic") and (fest_feature3=1){fest_feature1=0;fest_feature2=0;}
+	            if (onceh=0) and (fest_feature3=0){
+					onceh=1;
+					fest_feature3=1;
+				}
+	            if (onceh=0) and (fest_feature3=1) and (fest_type!="Chapter Relic"){
+					onceh=1;
+					fest_feature3=0;
+				}
+	            if (fest_type="Chapter Relic") and (fest_feature3=1){
+					fest_feature1=0;
+					fest_feature2=0;
+				}
 	        }
         
 	        // Always at least one feature
 	        if (fest_type!="Triumphal March") and (fest_type!="Chapter Sermon"){
 	            if (fest_feature1=0) and (fest_feature2=0) and (fest_feature3=0) then fest_feature1=1;
 	        }
         
-	        // Attendants
+	        // TODO Attendants 
```

Comment:
future PR?

Reply:
Another PR would ideally be the whole restructure of the celebrations/parades you do in the chapter. But would need to re read lore to see how this stuff was handled and probably varies with each chapter somewhat